ingestion/src/metadata/ingestion/source/database/databricks.py

```diff
@@ -210,31 +210,32 @@ def get_database_names(self) -> Iterable[str]:
         else:
             results = self.connection.execute("SHOW CATALOGS")
             for res in results:
-                new_catalog = res[0]
-                database_fqn = fqn.build(
-                    self.metadata,
-                    entity_type=Database,
-                    service_name=self.context.database_service.name.__root__,
-                    database_name=new_catalog,
-                )
-                if filter_by_database(
-                    self.source_config.databaseFilterPattern,
-                    database_fqn
-                    if self.source_config.useFqnForFiltering
-                    else new_catalog,
-                ):
-                    self.status.filter(database_fqn, "Database Filtered Out")
-                    continue
-
-                try:
-
-                    self.set_inspector(database_name=new_catalog)
-                    yield new_catalog
-                except Exception as exc:
-                    logger.error(traceback.format_exc())
-                    logger.warning(
-                        f"Error trying to process database {new_catalog}: {exc}"
+                if len(res) > 0:
+                    new_catalog = res[0]
```

Comment:
Is this will always be an `str` type? And position 0 will be the database name?

Reply:
yes it will be str and database name.